docs/development.md

```diff
@@ -11,6 +11,9 @@ NOTE: There is an issue with bazel 0.6.x. As a workaround, use 0.5.x, or pass
 the flag `--incompatible_comprehension_variables_do_not_leak=false` to bazel
 0.6.x invocations.
 
+To push an image to Google Container registry you'll also have to have [docker-credential-gcr](https://github.com/GoogleCloudPlatform/docker-credential-gcr#building-from-source) installed and configured. This allows  
```

Comment:
There is no need to build this from source as it can be installed via the `gcloud components install docker-credential-gcr` command or just simply by downloading the binary via https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases so we shouldn't link to building from source.

Also, we should instruct users that they will need to subsequently `docker-credential-gcr configure-docker` after installing or just point them to the installation and usage section https://github.com/GoogleCloudPlatform/docker-credential-gcr#installation-and-usage to perform that command.